.github/dependabot.yml

```diff
@@ -14,3 +23,41 @@ updates:
       - dependency-name: "third_party/pigweed/repo"
       - dependency-name: "third_party/mbedtls/repo"
       - dependency-name: "third_party/openthread/repo"
+
+      - dependency-name: "third_party/ameba/repo"
```

Comment:
we may want to check with platform owners on here as well.

For several I believe we generally do want updates (e.g. editline, imgui, inipp...) however I am unsure if platform owners want to manage their own updates.

Will ping SWTT slack.

Reply:
Nobody has complained, and one supported, let's give this a shot.